### ISSUE TITLE
Unable to deploy Lambdas with a DynamoDB stream as the event source

### ISSUE THREAD

**Issue by Jaystified** [localstack/localstack#1620]
With the following serverless.yml file:
```yml
service: localtest

provider:
  name: aws
  runtime: nodejs10.x

functions:
  hello:
    handler: handler.hello
    events:
    - stream:
        type: dynamodb
        arn:
          Fn::GetAtt: [fooTable, StreamArn]

resources:
  Resources:
    fooTable:
      Type: AWS::DynamoDB::Table
      Properties:
        TableName: fooTable
        AttributeDefinitions:
          - AttributeName: foo
            AttributeType: S
        KeySchema:
          - AttributeName: foo
            KeyType: HASH
        ProvisionedThroughput:
          ReadCapacityUnits: 1
          WriteCapacityUnits: 1
        StreamSpecification:
          StreamViewType: NEW_AND_OLD_IMAGES

plugins:
- "serverless-localstack"

custom:
  localstack:
    stages:
      - "local"
```
It fails with a generic error (500) on the deployment end and with `:ERROR:localstack.services.cloudformation.cloudformation_starter: Unable to parse and create resource "HelloEventSourceMappingDynamodbFooTable": 'NoneType' object has no attribute 'function_arn'` on the localstack end.
It deploys fine to AWS.

**Comment by whummer** [comment]
Thanks for reporting @Jaystified . This should be fixed in #1628 - can you please give it a try with the latest version of the Docker image. Thanks

**Comment by Jaystified** [comment]
Can confirm it works now. Thanks!
